src/azure-cli/azure/cli/command_modules/resource/_params.py

```diff
@@ -820,6 +827,17 @@ def load_arguments(self, _):
         c.argument('file', arg_type=bicep_file_type, help="The path to the ARM template to decompile in the file system.")
         c.argument('force', arg_type=bicep_force_type, help="Allows overwriting the output file if it exists.")
 
+    with self.argument_context('bicep decompile-params') as c:
+        c.argument('file', arg_type=bicep_file_type, help="The path to the parameters file to build in the file system.")
+        c.argument('bicep_file', arg_type=CLIArgumentType(options_list=['--bicep-file'],
+                                                          completer=FilesCompleter(),
+                                                          type=file_type,
+                                                          help="Path to the bicep template file (relative to the .bicepparam file) that will be referenced in the using declaration."))
```

Comment:
We usually define `CLIArgumentType` when the code needs to be reused
```suggestion
        c.argument('bicep_file', completer=FilesCompleter(), type=file_type, help="Path to the bicep template file (relative to the .bicepparam file) that will be referenced in the using declaration.")
```